include/rocksdb/options.h

```diff
@@ -1356,6 +1356,12 @@ struct ReadOptions {
   // processing a batch
   std::chrono::microseconds deadline;
 
+  // It limits the maximum cumulative value size of the keys in batch while
+  // reading through MultiGet. Once the cumulative value size exceeds this value
+  // then all the remaining keys are returned with status Aborted.
+  // Defualt: 0.
```

Comment:
If you keep 0 as a special value, this comment needs to mention that.

Although our internal checks would be more verbose, I suggest instead using std::numeric_limits<uint64_t>::max() so that the behavior is continuous across the range of settings. (Though this case isn't as weird as -1 for files_size_error_margin.)

Maybe a more descriptive name like "value_size_soft_limit"? I could foresee us adding a hard "value_size_limit" that would apply to Get and MultiGet. A soft limit (allowed to exceed but none after that) obviously doesn't make sense for Get.

Reply:
Thank you.  Yes it makes sense to be consistent with different range of settings. I will make the changes.